backend/donor-service/configs/config.js

```diff
@@ -32,6 +32,9 @@ const ESIGN_STATUS = Object.freeze({
 const ESIGN_VALIDATION_KAFKA_BROKERS = process.env.ESIGN_VALIDATION_KAFKA_BROKERS || 'localhost:5101';
 let ESIGN_VALIDATION_EXPIRE_TIME = process.env.ESIGN_VALIDATION_EXPIRE_TIME || 2*60;
 const ESIGN_VALIDATION_PREVENT_3RD_PARTY = process.env.ESIGN_VALIDATION_PREVENT_3RD_PARTY === "true" || false;
+const METRICS_URL = process.env.METRICS_URL || 'http://metrics:8070';
+const KAFKA_BROKER = process.env.KAFKA_BROKER || 'kafka:9092';
```

Comment:
this can be renamed to know for what purpose it is

Reply:
Changed it to KAFKA_BROKER_FOR_METRICS_SERVICE